presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2382,6 +2382,26 @@ public void dropMaterializedView(ConnectorSession session, SchemaTableName viewN
         }
     }
 
+    @Override
+    public List<String> getValidRefreshMaterializedViewFilterColumns(ConnectorSession session, SchemaTableName viewName)
+    {
+        requireNonNull(viewName, "viewName is null");
+
+        Optional<Table> viewTable = metastore.getTable(viewName.getSchemaName(), viewName.getTableName());
```

Comment:
This interface is out of date. We need `MetastoreContext` now. Please rebase.

Reply:
The most recent check-in should use the MetastoreContext arg.